scanner/metadata/metadata.go

```diff
@@ -238,18 +245,45 @@ func (t Tags) Lyrics() string {
 	return string(res)
 }
 
-func (t Tags) getGainValue(tagName string) float64 {
+func (t Tags) getGainValue(gainType GainType) float64 {
```

Comment:
I rather have explicit tag names in the parameters, instead of constants. Can you change this to:
```suggestion
func (t Tags) getGainValue(rgTagName, r128TagName string) float64 {
```